arangodb-net-standard/IndexApi/IIndexApiClient.cs

```diff
@@ -37,13 +39,85 @@ internal interface IIndexApiClient
+        Task<IndexResponseBase> PostFulltextIndexAsync(PostIndexQuery query,
+            PostFulltextIndexBody body,
+            CancellationToken token = default);
+
+        /// <summary>
+        /// Creates a new geo-spatial index
+        /// </summary>
+        /// <param name="query">Query parameters for the request.</param>
+        /// <param name="body">The properties of the new index.</param>
+        /// <param name="token">A CancellationToken to observe while waiting for the task to complete or to cancel the task.</param>
+        /// <returns></returns>
+        Task<IndexResponseBase> PostGeoSpatialIndexAsync(PostIndexQuery query,
+            PostGeoSpatialIndexBody body,
+            CancellationToken token = default);
+
+        /// <summary>
+        /// Creates a new hash index
+        /// </summary>
+        /// <param name="query">Query parameters for the request.</param>
+        /// <param name="body">The properties of the new index.</param>
+        /// <param name="token">A CancellationToken to observe while waiting for the task to complete or to cancel the task.</param>
+        /// <returns></returns>
+        Task<IndexResponseBase> PostHashIndexAsync(PostIndexQuery query,
+            PostHashIndexBody body,
+            CancellationToken token = default);
+
+        /// <summary>
+        /// Creates a new multi-dimensional index
+        /// </summary>
+        /// <param name="query">Query parameters for the request.</param>
+        /// <param name="body">The properties of the new index.</param>
+        /// <param name="token">A CancellationToken to observe while waiting for the task to complete or to cancel the task.</param>
+        /// <returns></returns>
+        Task<IndexResponseBase> PostMultiDimensionalIndexAsync(PostIndexQuery query,
+            PostMultiDimensionalIndexBody body,
+            CancellationToken token = default);
+
+        /// <summary>
+        /// Creates a new persistent index
+        /// </summary>
+        /// <param name="query">Query parameters for the request.</param>
+        /// <param name="body">The properties of the new index.</param>
+        /// <param name="token">A CancellationToken to observe while waiting for the task to complete or to cancel the task.</param>
+        /// <returns></returns>
+        Task<IndexResponseBase> PostPersistentIndexAsync(PostIndexQuery query,
+            PostPersistentIndexBody body,
+            CancellationToken token = default);
+
+        /// <summary>
+        /// Creates a new skiplist index
+        /// </summary>
+        /// <param name="query">Query parameters for the request.</param>
+        /// <param name="body">The properties of the new index.</param>
+        /// <param name="token">A CancellationToken to observe while waiting for the task to complete or to cancel the task.</param>
+        /// <returns></returns>
+        Task<IndexResponseBase> PostSkiplistIndexAsync(PostIndexQuery query,
```

Comment:
Skiplist and hash indexes shows as deprecated since 3.9, should we mark the endpoints as Obsolete? `skiplist` and `hash` are aliases for `persistent` since 3.7 but in the HTTP API they are deprecated later in 3.9:

https://www.arangodb.com/docs/3.9/http/indexes.html#hash-index

Reply:
I've marked PostSkiplistIndexAsync() and PostHashIndexAsync() as obsolete.